definitions/variable/technology/technologies.yaml

```
@@ -4,6 +4,12 @@
 - Capacity|Electricity|{Electricity Input}:
     description: Total installed electricity generation capacity from {Electricity Input}
     unit: GW
+- Capacity|Electricity|Energy Storage System:
+    description: Total installed electricity generation capacity from all {Energy Storage System Type} storage systems
```

If the tag is not used in the variable name, it does not make sense to use it in the description
```suggestion
    description: Total installed electricity generation capacity from all storage systems
```